js/Components/Headline/rolling.js

```diff
@@ -0,0 +1,42 @@
+import { constants } from "../../Data/constants.js";
+
+const timer = [];
+
+const repeatRolling = (rollingElement) => {
+  rollingElement.style.transitionDuration =
+    constants.ROLLING_TRANSITION_DURATION_MS + "ms";
+  rollingElement.style.marginTop = "-16px";
+
+  window.setTimeout(() => {
```

Comment:
이 부분 동일하게 맞춰주면 좋을것 같아요~

```suggestion
  setTimeout(() => {
```